server/eventlog.go

```diff
@@ -88,17 +89,44 @@ func parsePCClientEventLog(rawEventLog []byte, pcrs *tpmpb.PCRs, loader Bootload
 	}, createGroupedError("failed to fully parse MachineState:", errors)
 }
 
-func parseCanonicalEventLog(rawCanonicalEventLog []byte, pcrs *tpmpb.PCRs) (*pb.MachineState, error) {
+// ParseCosCanonicalEventLogPCR takes in a raw COS CEL and PCR banks, validates and returns it's
```

Comment:
nit: ParseCosCELPCR. No need to spell out the entire Canonical Event Log

Reply:
Why don't we return a *pb.AttestedCosState?

Reply:
and on the comment: ParseCosCELPCR takes an encoded COS CEL and PCR bank, replays the CEL against the PCRs, and returns the AttestedCosState

Reply:
Change parameter to register.PCRBank,

Keep the return type for now, will change it in a later PR

Reply:
A later PR will require both a breaking change and refactoring in the attestation service. Why not now?